data/hai/hai reveal 1 intro.txt

```diff
@@ -827,23 +829,24 @@ mission "Hai Leaks Response 4A"
 		personality staying marked mute
 		system "Waypoint"
 		ship "Geodesic Geocoris (Phylactery)" "Phylactery"
-		dialog `After successfully boarding the Geocoris and defeating its pirate crews, you are greeted by the relieved captive Hai. You promise to escort their ship safely to Allhome. (You will need to swap the Geocoris to your flagship, and depart and land again in order to progress the missions properly.)`
+		dialog `After successfully boarding the Geocoris and defeating its pirate crews, you are greeted by the relieved captive Hai. You send crew aboard to fly the ship and promise to return them safely to Allhome.`
 #	on fail
 #		dialog `You have failed to recover the Geocoris. This is not critical, but does have long-term consequences. If you want the most complete ending, revert to the autosave or another earlier snapshot of the game.`
 #Too hand-holdy for present implementation and content.
 
 
 # Must be able to resolve earlier than the rest in this set.
-#The Geodesic Geocoris may become available to purchase specially later on.
+# The Geodesic Geocoris may become available to purchase specially later on.
 mission "00 Hai Leaks Response 4B"
 	landing
 	source "Allhome"
+	to offer
+		has "ship model (all): Geodesic Geocoris"
 	on offer
-		require "Failed Control System"
-		log `Successfully rescued the kidnapped Hai and reclaimed the Geocoris. Something is different about it.`
+		log `Successfully rescued the kidnapped Hai and reclaimed the Geocoris. This ship has been modified in ways that aren't of Hai origin, and which don't appear to be fully functional.`
 		event "returned geocoris" 365
 		conversation
-			`You have successfully rescued the kidnapped Hai and reclaimed the Geocoris. Something is different about it...`
+			`You have successfully rescued the kidnapped Hai and reclaimed the Geocoris. On your way back to <planet> the crew you sent across to fly it home report that something is different about it... It appears that it has been modified with systems of some kind which are not of Hai design. It also appears that not everything about those systems is perhaps working wholly as intended.`
```

Comment:
```suggestion
			`You have successfully rescued the kidnapped Hai and reclaimed the Geocoris. On your way back to <planet> the crew you sent across to fly it home report that something is different about it... It appears that it has been modified with some kind of systems which are not of Hai design. It also appears that perhaps not everything about those systems is working wholly as intended.`
```

Small style rearrangement.

Reply:
Relocating perhaps makes sense to me, but the other change doesn't read right to me. 
I don't think that's a word-order that would be allowed with "system*s*", but it's not a single system which would make it inappropriate to de-pluralise it.

Reply:
There's nothing improper in the rearranged word order, there is a singular kind which describes multiple systems not of Hai design.

With the original wording, it would actually need switched to "some kind which is not of Hai design," as it is referring to multiple systems of one kind which is not of Hai design. Of the two, to me, the correction to the original was more awkward then the rearranging.

Reply:
"multiple systems of one kind" does not make the 'one kind' the item of pluralisation. The subject is still the 'multiple systems' ... 'which _are_ not of Hai design'.

Reply:
You are incorrect.

This is a specific application of the adjective phrase "some kind of/of some kind" - the order of which is determined by whether it precedes or follows the subject noun. There are a range of similar phrases that have similar behaviour and that behaviour is knowledge-dependent.

In this context the word "some" is being used to mean that you don't know "which particular kind."
The word order can be approached in a number of different ways usually, however:
- "some kind of systems" is the same form as "those kind of systems" or "that kind of systems"
- - the first one requires the knowledge that there are multiple countable systems of a kind - knowledge which you don't have
- - the second one is never correct
- "some kind*s* of systems" is the same form as "those kind*s* of systems" 
- - this specifically communicated that there are multiple different types of systems - which again is knowledge you don't have *and* only works with a subsequent grouping noun associative clause such as "on the ship".

Consequently the form preceding the noun can only be applied to a singular form noun, because when using "some" in this form there is no way to possess the knowledge required to identify the nature of the subject.
From the in-game context, this is an attempt to recreate a "System*s* core" which has multiple systems. We know it is not a single system, so it must be the plural noun.
As such it can only be applied in the "of some kind" form following the noun.

This makes "of some kind", taken together, an attributive phrasal adjective and so cannot ever be the subject.

Subsequently we have, "which are not of Hai design."
This is a relative clause. And, as just established, it is referring to a plural noun which cannot be depluralised.
A relative clause connects ideas by using pronouns that relate to something previously mentioned and allows the writer to combine two independent clauses into one sentence. A relative clause is also known as an adjective clause.
In this case it has the following components:
- "which" (relative pronoun)
- "are" (verb form relating to plural noun)
- "not of Hai design" (compound adjectival phrase containing the adjectival prepositional modifier indicating the possessive relationship "not of", and the attributive noun "Hai design")

Commonly there is a comma preceding a relative clause when the preceding sentence contains any compound elements, and so in deference to the concern a comma can go after "kind".

Relevant Sources:
https://www.javatpoint.com/adjective-phrase
https://english.stackexchange.com/questions/111666/some-kind-of-singular-or-plural
https://gohighbrow.com/the-passive-relative-clauses/
[https://academicguides.waldenu.edu/writingcenter/grammar/clauses](https://academicguides.waldenu.edu/writingcenter/grammar/clauses#:~:text=A%20relative%20clause%20connects%20ideas,relative%20clauses%3A%20restrictive%20and%20nonrestrictive.)
https://learningenglish.voanews.com/a/everyday-grammar-when-nouns-act-like-adjectives/2998821.html

Since I was here anyway, I did a bit of a check regarding the placement of "perhaps".
The following forms are apparently all valid:
1. It also appears that perhaps not everything about those systems is working wholly as intended.
2. It also appears that everything about those systems is not perhaps working wholly as intended.
3. It also appears that not everything about those systems is perhaps working wholly as intended.

However, if the objective is readability it would appear in this context that it shouldn't be separated from the "not", and that the "not" shouldn't be separated from the "everything" for maximum comprehensibility from word order alone.
So you were indeed most correct on that point.

```suggestion
			`You have successfully rescued the kidnapped Hai and reclaimed the Geocoris. On your way back to <planet> the crew you sent across to fly it home report that something is different about it... It appears that it has been modified with systems of some kind, which are not of Hai design. It also appears that perhaps not everything about those systems is working wholly as intended.`
```